### DISCUSSION TITLE
My Github Profile

### DISCUSSION THREAD

**Discussion by Sagar-Sharma-7** [community/community#33562] [Other Feature Feedback, Questions, & Ideas] [answered]
How's my GitHub profile?

**Comment by radar-blip** [comment]
It exists. 💯

**Reply by samuel-ma** [reply]
Your so cold @linuxlawson😂
And also somehow on the answer section of every discussion
With a Platinum Galaxy Brain, I think you should honestly just quit responding to these discussions
Let other small fish get to feast as well.....

**Comment by Matteo-it** [comment, marked as answer]
Awesome!

**Comment by luisestebanas** [comment]
Original 👌

**Comment by Dmitriy-Tyukh** [comment]
Original
